public/Day 178/script.js

```
@@ -0,0 +1,970 @@
+            }
+            addEvent('Massive flood covered low-lying areas!');
+            break;
+        case 2:
+            // Drought
+            water -= 20;
+            food -= 30;
+            addEvent('Severe drought reduced water and food supplies!');
+            break;
+        case 3:
+            // Meteor strike
+            let centerX = Math.floor(Math.random() * CANVAS_WIDTH);
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)